web/src/admin/providers/ldap/LDAPProviderViewPage.ts

```diff
@@ -122,13 +122,7 @@ export class LDAPProviderViewPage extends AKElement {
             return html``;
         }
         return html`
-            ${
-                this.provider?.outpostSet.length < 1
-                    ? html`<div slot="header" class="pf-c-banner pf-m-warning">
-                          ${msg("Warning: Provider is not used by any Outpost.")}
-                      </div>`
-                    : html``
-            }
+            ${this.provider?.outpostSet.length < 1 ? html`<div slot="header" class="pf-c-banner pf-m-warning">${msg("Warning: Provider is not used by any Outpost.")}</div>` : html``}
```

Comment:
Prettier having a weird day.  Not sure why it's not reformatting this.  It's harmless, but...